ui/overview.js

```diff
@@ -130,7 +130,7 @@ function toggleApps(overview) {
         return;
     }
 
-    if (!WorkspaceMonitor.hasActiveWindows()) {
+    if (!ignoreNoWindowsDialog && !WorkspaceMonitor.hasActiveWindows()) {
```

Comment:
You may want to update your WorkspaceMonitor implementation with this: https://github.com/endlessm/eos-desktop-extension/blob/master/ui/workspaceMonitor.js#L87

Reply:
Not sure I get what you mean, care to ellaborate?

Reply:
The implementation of WorkspaceMonitor in this extension still uses `Shell.AppSystem`, which is unreliable. The URL I sent shows the more recent and reliable implementation, which inspects the window group tree.

Reply:
I see, I changed (still local) the panel impl to use the window group to detect whether there are visible windows but it still uses AppSystem to check if there are active apps (AppSystem.get_running() > 0) which is what that code path uses. Do you think we should check for visible windows here? I guess not, just want to verify.

Reply:
It makes sense to use the same check in both cases. If an app has no window, GNOME Shell won't be tracking it anyway.

Reply:
You mean to use getActiveApps() or getVisibleApps()? I think we want getActiveApps() (as it is now) here because we want to show the window switcher even if all apps are minimized and getVisibleApps() will return false if all apps are minimized...

Reply:
Updated the workspaceMonitor impl but still using getActiveApps() in this codepath, please check.